app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,63 @@
+require 'pry'
+class RentalsController < ApplicationController
+
+  def index
+    rentals = Rental.all
+    render json: rentals.as_json(only: [:id, :customer_id, :movie_id]), status: :ok
+  end
+
+  def checkout
+
+    # check_out = Date.today.to_s
+    # due_date = (Date.today+7).to_s
+    #
+    # movie_id = params[:movie_id].to_i
+    # movie = Movie.find_by(id: movie_id)
+    movie = Movie.find_by(id: rental_params[:movie_id])
+    customer = Customer.find_by(id: rental_params[:customer_id])
+    # customer_id = params[:customer_id].to_i
+    # customer = Customer.find_by(id: customer_id)
+    if movie == nil || customer == nil
+      render json: {errors: "No movie checked out, missing movie or customer"}, status: :bad_request
+    else
+      if movie.available_inventory > 0
+        rental = Rental.new(rental_params)
+        if rental.save
+          inventory = movie.calculate_available_inventory()
+          movie.save_available_inventory(inventory)
+          customer[:movies_checked_out_count] += 1
```

Comment:
Could you make a method that increments the `customer`'s checked out count field instead of operating on it directly here?